docs/tutorial/index.md

```diff
@@ -8,10 +8,18 @@ Congratulations! You have started the container for this tutorial!
 Let's first explain the command that you just ran. In case you forgot,
 here's the command:
 
+```cli
+docker run -d -p 80:80 allanl95/getting-started-bootdotdev
+```
+
+You can try dockers official image by running:
```

Comment:
Meh, why even confuse them is my question? I'd just say something like:

Run our fork of Docker's getting started image (our fork adds CORS headers so our tests can connect):

```
docker run -d -p 80:80 allanl95/getting-started-bootdotdev
```

Then I wouldn't even give them the URL of the original or have the note